lib/inertia_phoenix/plug.ex

```diff
@@ -49,4 +50,17 @@ defmodule InertiaPhoenix.Plug do
     |> put_status(:conflict)
     |> halt()
   end
+
+  def check_redirect(conn) do
+    conn
+    |> Plug.Conn.register_before_send(fn conn ->
+      case(conn.method in ["PUT", "PATCH", "DELETE"] and conn.status in [301, 302]) do
```

Comment:
IMHO an `if` statement would work better here :)

Reply:
thanks @zimt28 I just merged it so I could write the tests.